packages/iov-ethereum/src/ethereumconnection.spec.ts

```diff
@@ -84,63 +99,130 @@ describe("EthereumConnection", () => {
     connection.disconnect();
   });
 
-  it("can post transaction", async () => {
-    pendingWithoutEthereum();
+  describe("postTx", () => {
+    let postedTxId: TxId;
```

Comment:
Tests should never depend on each other. This allows you to [randomize](https://jasmine.github.io/api/3.3/Configuration.html) [the oder](https://github.com/w3c/respec/issues/1552) and [focus](https://jasmine.github.io/2.1/focused_specs.html) or [exclude](https://jasmine.github.io/2.0/introduction.html#section-Pending_Specs) tests one by one.

It is better to just duplicate the posting

Reply:
Yes, you are right. Anyway..this is a work around, we should get a ganache database loaded with multiple transactions and blocks, such database can be installed from iov github